docusaurus.config.js

```diff
@@ -124,7 +124,7 @@ module.exports = {
           anonymizeIP: true,
```

Comment:
Google Analytics 4 (GA4) replaced Google Universal Analytics (UA) effective 1 July 2023. See https://support.google.com/analytics/answer/11583528, so you can drop the UA ID on line 123 above (well, remove the entry from 122-125). /cc @jvanz